src/sentry/south_migrations/0328_backfill_release_stats.py

```diff
@@ -0,0 +1,895 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models, transaction
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Deploy = orm['sentry.Deploy']
+        Release = orm['sentry.Release']
+        ReleaseCommit = orm['sentry.ReleaseCommit']
+
+        queryset = Release.objects.all()
+        for release in RangeQuerySetWrapperWithProgressBar(queryset):
+            with transaction.atomic():
+                last_deploy = Deploy.objects.filter(
+                    release=release,
+                ).order_by('-date_finished').first()
+                last_commit = ReleaseCommit.objects.filter(
+                    release=release,
+                ).order_by('order').first()
+                Release.objects.filter(
+                    id=release.id,
+                ).select_for_update().update(
```

Comment:
would be kind of nice to do this as part of RangeQuerySetWrapper, but could potentially be very expensive